dt-assets/js/comments.js

```diff
@@ -419,13 +593,14 @@ jQuery(document).ready(function($) {
         if (text.includes("http") && !url.includes("http")){
           [url, text] = [text, url]
         }
-        url = url.includes('http') ? url : `${window.wpApiShare.site_url}/${window.wpApiShare.post_type}/${url}`
-        return `<a href="${url}">${text}</a>`
+        url = url.includes("http")
+          ? url
+          : `${window.wpApiShare.site_url}/${window.wpApiShare.post_type}/${url}`;
+        return `<a href="${url}">${text}</a>`;
```

Comment:
@Filmscore88 Is your IDE automatically reformatting these lines?

Reply:
Yes, but I switched to Visual Studio to avoid that.